packages/tools/client-debugger/client-debugger/src/test/ClientDebugger.test.ts

```diff
@@ -26,9 +29,32 @@ describe("ClientDebugger unit tests", () => {
 	const otherContainerId = "test-container-id-other";
 	let otherContainer: IContainer | undefined;
 
+	const registry: DebuggerRegistry = getDebuggerRegistry();
+
+	let clientId: string | undefined;
+
+	let debuggerRegistered = false;
+	let debuggerClosed = false;
+	let audienceAdded = false;
+	let audienceRemoved = false;
+
 	beforeEach(async () => {
```

Comment:
You'll want to add an `afterEach` hook to ensure all of the boolean state we're validating against is reset to `false` between tests.

Reply:
Added!